src/dev-server/index.ts

```diff
@@ -44,6 +45,8 @@ export default class DevServer {
   // returns a promise that resolves when the dev server is stopped
   async start(): Promise<void> {
     const { watch } = this.opts
+
+    console.log('Starting dev server...')
```

Comment:
should this console log be here?

Reply:
Yeah, one of the things I was noticing is that when the dev server started up it looked like it wasn't working because it was doing things like starting the dev server and building the docker image, etc. And those things (really, the docker image build) took awhile but weren't showing any output. So I wanted to print out a few statements that let the user know that the dev command is working.